cmd/skopeo/copy.go

```diff
@@ -14,11 +14,11 @@ func copyHandler(context *cli.Context) error {
 		return errors.New("Usage: copy source destination")
 	}
 
-	policyContext, err := getPolicyContext(context)
+	trustContext, err := getTrustContext(context)
 	if err != nil {
-		return fmt.Errorf("Error loading verification policy: %v", err)
+		return fmt.Errorf("Error loading trust verification: %v", err)
```

Comment:
I agree,  I was just throwing this together for discussion.  I liked this slightly better the 
Error loading verification trust.

"Error loading trust policy" is probably ok.